conf/config.yaml

```diff
@@ -65,6 +65,11 @@ nginx_config:                     # config for render the template to genarate n
     client_header_timeout: 60s     # timeout for reading client request header, then 408 (Request Time-out) error is returned to the client
     client_body_timeout: 60s       # timeout for reading client request body, then 408 (Request Time-out) error is returned to the client
     send_timeout: 10s              # timeout for transmitting a response to the client.then the connection is closed
+  server:                          # set custusmized directives in nginx.conf server field  
+    client_body_buffer_size: 16k   # Sets buffer size for reading client request body
+  #  server_tokens: on             # Enables emitting nginx version on error pages and in the "Server"
+  #  add_header: "API-Gateway-ID: '001'" # add custusmized header
+  #  underscores_in_headers: on    # Enable the use of underscores in client request header fields
```

Comment:
I think you can delete these three lines, they will mislead the users, in fact, we do not recommend users to use these three items at all.

Reply:
here I just want remind that they can set a global header

Reply:
user should use the plugin to set header instead of Nginx configure.
We should not let users perceive the existence of Nginx.

Reply:
I will drop this line of demo code